docs/table.rst

```diff
@@ -394,11 +411,11 @@ to make a note on how template caching works then,
  - values are encapsulated for named fields.
  - values are concatinated by anonymous fields.
  - <td> properties are stacked
- - last formatter will convert array with td properties into an actual tag.
+ - last decorator will convert array with td properties into an actual tag.
 
 Header and Footer
 -----------------
-When using with multiple formatters, the last formatter gets to render Header column.
+When using with multiple decorators, the last decorator gets to render Header column.
```

Comment:
Really last decorator is responsible of header cell? I thought first decorator do that and last decorator maybe formats footer cell ... Please check this - sounds strange.